paging_test.py

```diff
@@ -3406,15 +3406,15 @@ def test_ttl_deletions(self):
         # Set TTL to all row
         for row in data:
             s = ("insert into paging_test (id, mytext, col1, col2, col3) "
-                 "values ({}, '{}', {}, {}, {}) using ttl 3;").format(
+                 "values ({}, '{}', {}, {}, {}) using ttl 15;").format(
                 row['id'], row['mytext'], row['col1'],
                 row['col2'], row['col3'])
             self.session.execute(
                 SimpleStatement(s, consistency_level=CL.ALL)
             )
         self.check_all_paging_results(data, 8,
                                       [25, 25, 25, 25, 25, 25, 25, 25])
-        time.sleep(5)
+        time.sleep(15)
```

Comment:
Besides incrementing the time, I like the idea of waiting exactly the TTL time. Perhaps we could make that more obvious by putting the TTL in a var, to be used both in the queries and in the sleep:
```python
ttl_seconds = 15
for row in data:
    s = ("insert into paging_test (id, mytext, col1, col2, col3) "
         "values ({}, '{}', {}, {}, {}) using ttl {}").format(
        row['id'], row['mytext'], row['col1'], row['col2'], row['col3'], ttl_seconds)
    self.session.execute(SimpleStatement(s, consistency_level=CL.ALL))
self.check_all_paging_results(data, 8, [25, 25, 25, 25, 25, 25, 25, 25])
time.sleep(ttl_seconds)
self.check_all_paging_results([], 0, [])
```

Reply:
+1